default.nix

```
@@ -2,7 +2,7 @@
 
 buildGoModule {
   pname = "oplogtoredis";
-  version = "3.3.0";
+  version = "3.4.0";
```

This might be more fitting for the scope of the change:

```suggestion
  version = "3.3.1";
```

---

I don't mean to battle over minutiae, but wouldn't this semantically be a minor version rather than a patch since it isn't a bugfix?

---

Patch version Z (x.y.Z | x > 0) MUST be incremented if only backward compatible bug fixes are introduced. A bug fix is defined as an internal change that fixes incorrect behavior.

Minor version Y (x.Y.z | x > 0) MUST be incremented if new, backward compatible functionality is introduced to the public API. It MUST be incremented if any public API functionality is marked as deprecated. It MAY be incremented if substantial new functionality or improvements are introduced within the private code. It MAY include patch level changes. Patch version MUST be reset to 0 when minor version is incremented.

^ yeah, idk. I guess the change kind of fix their definition of a `bug`, which explains why that number would by called a patch version to me now. But, right, whatever you wish to go with, I'm fine with the minor change as well.

---

I really don't have much preference; I think it doesn't matter particularly much. I will leave it just because it's easier if you don't mind.